src/main/java/seedu/address/ui/DetailPanel.java

```diff
@@ -81,10 +80,12 @@ public void updateDetails() {
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tagDetails.getChildren().add(new Label(tag.tagName)));
 
-        meetings.setText("Meetings\t:\n" + "(coming in v1.4)");
-        // when it's ready, put person.getMeetings()
+        meetings.setText("Meetings\t:\n" + (person.getMeetings().isEmptyMeetings()
+                ? "You don't have a meeting arranged with this Udder"
```

Comment:
Would it be better to save the String here as a private static variable i.e. MESSAGE_NO_MEETINGS?